src/lib/session_table.c

```diff
@@ -3,13 +3,15 @@
  * Copyright (c) 2018, Intel Corporation
  * All rights reserved.
  */
+#include <stdio.h>
```

Comment:
why did you add stdio?

Reply:
Sorry about that. Leftover from some debug logging.